mathics/doc/common_doc.py

```diff
@@ -1143,20 +1143,17 @@ def doc_part(self, title, modules, builtins_by_module, start):
         # packages inside ``mathics.builtin``.
         modules_seen = set([])
 
-        want_sorting = True
-        if want_sorting:
-            module_collection_fn = lambda x: sorted(
+        def module_collection_fn(_) -> list:
```

Comment:
why does this function need to have an argument that in the end is not used?

Reply:
You are right. This 

This is a holdover from the stuff that came before where the "module" parameter was passed in but later it was taken from by the closure from a surrounding variable. 

Is removed. now.